riotfile.py

```diff
@@ -154,7 +154,7 @@ def select_pys(min_version=MIN_PYTHON_VERSION, max_version=MAX_PYTHON_VERSION):
             name="benchmarks",
             pys=select_pys(),
             pkgs={"pytest-benchmark": latest, "msgpack": latest},
-            command="pytest {cmdargs} tests/benchmarks",
+            command="pytest --benchmark-warmup=on --benchmark-autosave {cmdargs} tests/benchmarks",
```

Comment:
do we want autosave?

will we just overwrite with each serial run of `pytest`? (this is the problem we have with junit.xml files, see `conftest.py`)

Reply:
The bigger picture is to collect that data along with whatever else we produce in CI and upload to an s3 bucket. File names will include commit hashes so there shouldn't be collisions

Reply:
If you do `riot run benchmark`, then running with py 3.7 will overwrite the file generated for py 3.6. so after the run you'll only have 1 file, when you should have 1 per-python version. 

this is why in `conftest.py` we dynamically change the name of the `junit.xml` to `junit.<pid>.xml` (probably should be a uuid in hindsight, but seems fine for now)

Reply:
Ah right, of course!

Reply:
@brettlangdon the default name had a timestamp so slim chance of collision, but I went for a custom name that includes a timestamp and the interpreter version info.

Reply:
oh cool, yeah, timestamp probably would have been enough, but 👍🏻 to including interpreter name!